packages/components/src/components/Snackbar/SnackbarProvider.tsx

```diff
@@ -11,6 +11,11 @@ type SnackbarContextType = {
   setOffset: (newOffset: number) => void
 }
 
+export const SNACKBAR_DURATIONS = {
```

Comment:
Don't feel strongly, but noting that with the rework this doesn't need to be a global const since now it's just a special hook-level override and if we opened duration up to consumers it'd be in snackbarOptions and also accessed just within the hook.

Reply:
Yeah I wasn't planning on opening up to consumers but exported it as a const so the values can also be available to `useSnackbar.tsx`.